editor_drafts/draft-prorock-spice-cose-sd-cwt.md

```diff
@@ -37,12 +37,9 @@ organization = "Transmute"
 
 .# Abstract
 
-This document describes how to perform selective disclosure of claims
-withing a CBOR Web Token (CWT) [@!RFC8392] as well as how to create and
-verify those tokens.
-
-This document does not define any new cryptography.
-
+This document describes a data minimization technique of use with CBOR Web Token (CWT) [@!RFC8392].
+The approach is based on SD-JWT, with changes to align with CBOR Object Signing and Encryption (cose).
+This document updates RFC8392.
```

Comment:
This is very debatable, but I feel that CWT should come with "batteries included", and I believe selective disclosure and related claims are just as fundamental to what a CWT is as the claims in the original RFC... I'd also like to avoid some of the challenges we've seen with SD-JWT, where its "not a JWT", but also sorta a JWT.